testutil/consul.go

```diff
@@ -56,3 +59,17 @@ func ConsulACLConfigFn(c *testutil.TestServerConfig) {
 	c.ACL.Tokens.InitialManagement = AdminToken
 	c.ACL.DefaultPolicy = "deny"
 }
+
+func GetHostAndPortFromAddress(address string) (string, int) {
```

Comment:
minor: This function wouldn't handle IPv6 IPs correctly. We could use https://pkg.go.dev/net#SplitHostPort instead (although I think we generally assume IPv4 in ECS and don't test with IPV6. But it is [supported](https://aws.amazon.com/about-aws/whats-new/2020/11/amazon-ecs-supports-ipv6-in-awsvpc-networking-mode/) in ECS).